js/samples/quickstart-csharp/Views/Shared/Error.cshtml

```diff
@@ -6,10 +6,10 @@
 <h1 class="text-danger">Error.</h1>
 <h2 class="text-danger">An error occurred while processing your request.</h2>
 
-@if (Model.ShowRequestId)
+@if (Model?.ShowRequestId ?? false)
 {
     <p>
-        <strong>Request ID:</strong> <code>@Model.RequestId</code>
+        <strong>Request ID:</strong> <code>@Model?.RequestId</code>
```

Comment:
May not need Model?. here since the only way to get here is if Model != null

Reply:
correct, done